web/ui/templates/_base.html

```diff
@@ -1,12 +1,12 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-        <meta name="robots" content="noindex,nofollow">
-        <title>Prometheus Time Series Collection and Processing Server</title>
-        <link rel="shortcut icon" href="{{ pathPrefix }}/static/img/favicon.ico?v={{ buildVersion }}">
-        <script src="{{ pathPrefix }}/static/vendor/js/jquery-3.3.1.min.js?v={{ buildVersion }}"></script>    
-        <script src="{{ pathPrefix }}/static/vendor/bootstrap-4.0.0/js/bootstrap.min.js?v={{ buildVersion }}"></script>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="robots" content="noindex,nofollow">
+    <title>{{ pageTitle }}</title>
+    <link rel="shortcut icon" href="{{ pathPrefix }}/static/img/favicon.ico?v={{ buildVersion }}">
+    <script src="{{ pathPrefix }}/static/vendor/js/jquery.min.js?v={{ buildVersion }}"></script>
+    <script src="{{ pathPrefix }}/static/vendor/bootstrap-3.3.1/js/bootstrap.min.js?v={{ buildVersion }}"></script>
```

Comment:
Same remark here.

Reply:
ill take a look at the build problems next week, apologies. its trying to load some bootstrap.scss and also not loading the correct type-ahead file for some reason.